[2021-12-01 16:47:44,288] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: FrequencyCount.map2 manual__2021-12-01T14:50:52.095578+00:00 [queued]>
[2021-12-01 16:47:44,300] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: FrequencyCount.map2 manual__2021-12-01T14:50:52.095578+00:00 [queued]>
[2021-12-01 16:47:44,301] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-01 16:47:44,301] {taskinstance.py:1242} INFO - Starting attempt 1 of 2
[2021-12-01 16:47:44,301] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-01 16:47:44,312] {taskinstance.py:1262} INFO - Executing <Task(MapOperator): map2> on 2021-12-01 14:50:52.095578+00:00
[2021-12-01 16:47:44,318] {standard_task_runner.py:52} INFO - Started process 75 to run task
[2021-12-01 16:47:44,324] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'FrequencyCount', 'map2', 'manual__2021-12-01T14:50:52.095578+00:00', '--job-id', '65', '--raw', '--subdir', 'DAGS_FOLDER/mapred.py', '--cfg-path', '/tmp/tmpwu87y3nn', '--error-file', '/tmp/tmpro4vj5kq']
[2021-12-01 16:47:44,324] {standard_task_runner.py:77} INFO - Job 65: Subtask map2
[2021-12-01 16:47:44,360] {logging_mixin.py:109} INFO - Running <TaskInstance: FrequencyCount.map2 manual__2021-12-01T14:50:52.095578+00:00 [running]> on host 1da30e33d42e
[2021-12-01 16:47:44,451] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=FrequencyCount
AIRFLOW_CTX_TASK_ID=map2
AIRFLOW_CTX_EXECUTION_DATE=2021-12-01T14:50:52.095578+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-01T14:50:52.095578+00:00
[2021-12-01 16:47:44,727] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=FrequencyCount, task_id=map2, execution_date=20211201T145052, start_date=20211201T164744, end_date=20211201T164744
[2021-12-01 16:47:44,774] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-01 16:47:44,803] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
